src/pages/PrivateNotes/PrivateNotesEditPage.js

```diff
@@ -63,7 +63,23 @@ function PrivateNotesEditPage({route, personalDetailsList, session, report}) {
 
     // We need to edit the note in markdown format, but display it in HTML format
     const parser = new ExpensiMark();
-    const [privateNote, setPrivateNote] = useState(parser.htmlToMarkdown(lodashGet(report, ['privateNotes', route.params.accountID, 'note'], '')).trim());
+    const [privateNote, setPrivateNote] = useState(
+        Report.getDraftPrivateNote(report.reportID) || parser.htmlToMarkdown(lodashGet(report, ['privateNotes', route.params.accountID, 'note'], '')).trim(),
```

Comment:
We forgot to trim the value acquired from `Report.getDraftPrivateNote`

Reply:
@DylanDylann can you spin up a PR to fix this? 🙏🏼

Reply:
@MariaHCD @robertKozik Sorry for the wait there. I am working on it now

Reply:
https://github.com/Expensify/App/pull/30357 PR is ready for review @robertKozik